packages/maskbook/src/modules/CryptoAlgorithm/interfaces/interface.ec.ts

```diff
@@ -12,8 +12,8 @@ type Verify = (
 type DeriveAES = (
     priv: EC_Private_JsonWebKey,
     pub: EC_Public_JsonWebKey,
-    aes: AESName,
-    length: 256,
+    aes?: AESName,
```

Comment:
Why

Reply:
call it with only two parameter

Reply:
improve this function parameter to options pattern

```
type DeriveAES = (options: DeriveAESOptions) => PromiseLike<AESJsonWebKey>
```

Reply:
Why can't you call it with all parameters required?